### PR TITLE
Adding Vespa.json

### PR DESCRIPTION
**Pull request by HarshMN2345**
Implemented Vespa Search
Associated issue #692

To utilize this functionality, follow these steps:

Start Vespa Instance:
Ensure you have a running Vespa instance. If not, refer to Vespa's official documentation for installation instructions.

Add JSON File to Swirl Configuration:
Insert the provided JSON file into your Swirl configuration file.

Restart Swirl:
Restart Swirl to apply the changes.

Now, you can start searching for Vespa content. Use the following command:

swirl search vespa "my query"

### CODE REVIEW

**Comment by HarshMN2345** [comment]
@sidprobstein this is the new PR i have added with the changes you suggested

File: SearchProviders/vespa.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  "url": "https://localhost:8080/search/",
+  "query_template": "{url}?query={query_string}&fields={fields}&hits={hits}",
+  "query_processors": ["AdaptiveQueryProcessor"],
+  "query_mappings": "",
+  "result_processors": [
+    "MappingResultProcessor",
+    "LenLimitingResultProcessor",
+    "CosineRelevancyResultProcessor"
+  ],
+  "response_mappings": "RESULTS=root_object.hits.hits, #FOUND=root_object.hits.found, #RETRIEVED=root_object.hits.retrieved",
```
Comment (sidprobstein) [inline SearchProviders/vespa.json:15]: Sorry @HarshMN2345 please remove the hash marks (#) it should be 
FOUND=root_object.hits.found, not #FOUND=...

**Comment by HarshMN2345** [comment]
@sidprobstein done

**Review by sidprobstein — APPROVED** [review]

**Comment by HarshMN2345** [comment]
@sid-swirl due to some irrevelents pulls this pr is compromised i am adding another pr with same file

**Pull request closed without merging**